src/controllers/usersController.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const userService = require('../services/users');
+
+const getAll = (req, res) => {
+  const users = userService.getUsers();
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const userId = Number(req.params.userId);
+
+  if (isNaN(userId)) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const foundUser = userService.getUser(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(foundUser);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = userService.createUser(name);
+
+  res.statusCode = 201;
+  res.send(newUser);
+};
+
+const remove = (req, res) => {
+  const userId = Number(req.params.userId);
+  const foundUser = userService.getUser(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  userService.remove(userId);
+
+  res.sendStatus(204);
+};
+
+const update = (req, res) => {
+  const userId = Number(req.params.userId);
+  const foundUser = userService.getUser(userId);
+  const { name } = req.body;
+
+  if (!name || isNaN(userId) || typeof name !== 'string') {
```

Comment:
Split the conditional statement into separate checks for better error handling and readability.